hapi-proto/src/main/proto/ScheduleCreate.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package proto;
+
+/*-
+ * ‌
+ * Hedera Network Services Protobuf
+ * ​
+ * Copyright (C) 2018 - 2020 Hedera Hashgraph, LLC
+ * ​
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * ‍
+ */
+
+import "BasicTypes.proto";
+import "Timestamp.proto";
+
+message ScheduleCreateTransactionBody {
+  bytes transactionBody = 1; // The transaction serialized into bytes that must be signed
+  Key adminKey = 2; // (optional) The Key which is able to delete the Scheduled Transaction (if tx is not already executed)
+  Signers signers = 3; // The accounts which are able to authorise and sign the scheduled transaction. This could be a Threshold list of accounts or a simple Account list. Serves as definition to who should sign this scheduled TX in order for it to be executed.
+  SignatureMap signatureMap = 4; // The signature map containing the signature of at least one account defined in the signers proto on transaction creation
+  bool executeImmediately = 5; // Whether to execute the transaction immediately when all required signatures are collected. Currently only executeImmediately=true is supported
+  Timestamp executeAtTime = 6; // The time at which the transaction will be attempted to be executed. Currently not supported.
+}
+
+message Signers {
```

Comment:
After our call tonight, we will define this protobuf.
For now, the `signers` or `signKey` properties are not validated and are subject to change.